cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -115,16 +126,23 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 			}
 			layers[layersLen-1] = vmPath
 			wopts.LayerFolders = layers
-
-			parent, err = uvm.CreateWCOW(ctx, wopts)
+			if utc != nil {
+				parent, err = uvm.CloneWCOW(ctx, wopts, utc)
+			} else {
+				parent, err = uvm.CreateWCOW(ctx, wopts)
+			}
 			if err != nil {
 				return nil, err
 			}
 		}
-		err = parent.Start(ctx)
+		if utc != nil {
+			err = parent.StartClone(ctx)
+		} else {
+			err = parent.Start(ctx)
+		}
```

Comment:
Similar to above, a personal style choice: can we move this logic into `Start` and then in there call `start` or `startClone`?